contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -160,13 +160,28 @@ open class ContourLayout(
   attrs: AttributeSet? = null
 ) : ViewGroup(context, attrs) {
 
+  /**
+   * Whether Contour should respect padding set on this layout as part of laying out its subviews.
+   * When set to `true`, padding will influence the alignment of subviews to the layout's inner edges
+   * When set to `false`, padding will be ignored in all layout calculations.
+   * @see ViewGroup.setPadding
```

Comment:
Might be useful to add a line explaining why does this value even exists

Reply:
Agreed, will add